test/dummy/config/environments/test.rb

```diff
@@ -29,6 +29,9 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
+  # run tests in a sorted order (and get rid of rails its lengthy deprecation warning)
+  config.active_support.test_order = :sorted
```

Comment:
Except for this project I'm always using rspec, so I wasn't really on top of things.

But for the reason stated above I would definitely prefer it to be `:random`